db/migrate/20170914235746_add_quantity_to_order_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddQuantityToOrderItems < ActiveRecord::Migration[5.1]
+  def change
+    add_column :order_items, :quantity, :integer
```

Comment:
What's the reasoning of doing this here and not as a method on the Order model or Cart Poro?